docs/project/2024/Layla.el/index.md

```diff
@@ -0,0 +1,85 @@
+# Project Name
+
+:::info 
+
+**Author**: El-Ghoul Layla \
+**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-Scarletwtc
+:::
+
+## Description
+
+The project revolves around a multi-storey parking lot. At the entrance, an infrared sensor will detect the presence of vehicles. After the detection of a vehicle a servomotor barrier will allow its entry. Infrared sensors will be implanted at each parking lot to monitor occupation.There will be green leds indicating available parking spaces, and red ones indicating occupancy.The second floor will be designated for authorized vehicles only.To make this happen, an RFID system will be utilized. In case an unauthorized vehicle tries to access, a buzzer will be activated alerting security personnel. When an authorized vehicle is detected, an LED light will turn on. The exit will also implement an infrared sensor and a servomotor barrier. A flame sensor will indicate the presence of fire, in case of fire a buzzer will be activated and the servomotors will automatically open to enable evacuation procedures.
+
+
+## Motivation
+
+These days, cars have become a necessity rather than a luxury. People rely on them for their everyday activities, whether it's going to work, running errands, or simply enjoying some leisure time. One major concern for them is finding a place to park wherever they go. Whether it's for work, fun, or shopping, having a convenient and safe parking space is crucial for their comfort and peace of mind.
+I picked this project because it addresses this concern. My project focuses on making parking lots safer, ensuring a better parking experience for everyone.
+
+## Architecture 
```

Comment:
The high level hardware and software architecture.

Do leave the pictures and text that you already have and add the required things.